apps/tools/tests/test_test_course.py

```diff
@@ -0,0 +1,55 @@
+import collections
+import json
+import os
+
+from librelingo_json_export.export import export_course
+from librelingo_yaml_loader.yaml_loader import load_course
+
+Settings = collections.namedtuple(
+    "Settings",
+    [
+        "dry_run",
+    ],
+)
+
+
+def test_course(tmpdir):
+    root = os.path.abspath(__file__)
+    root = os.path.dirname(root)
+    root = os.path.dirname(root)
+    root = os.path.dirname(root)
+    root = os.path.dirname(root)
+    print(root)
+
+    path_to_course = os.path.join(root, "courses", "test")
```

Comment:
I am thinking, this path could be moved to a pytest fixture, for example a fixture called `test_course_snapshot`

Reply:
but why, is in needed in other places as well?